api/src/pcapi/alembic/versions/20240506T084310_571e971b5c73_drop_boost_username_and_password.py

```diff
@@ -0,0 +1,23 @@
+"""Drop "username" & "password" columns from "boost_cinema_details" table.
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "571e971b5c73"
+down_revision = "400f736caab1"
+branch_labels: tuple[str] | None = None
+depends_on: list[str] | None = None
+
+
+def upgrade() -> None:
+    op.drop_column("boost_cinema_details", "password")
+    op.drop_column("boost_cinema_details", "username")
+
+
+def downgrade() -> None:
+    op.add_column("boost_cinema_details", sa.Column("username", sa.TEXT(), autoincrement=False, nullable=False))
+    op.add_column("boost_cinema_details", sa.Column("password", sa.TEXT(), autoincrement=False, nullable=False))
```

Comment:
Ok pour la nouvelle migration pré-déploiement. Par contre, dans la migration **post**-déploiement, le downgrade ne doit pas remettre la contrainte `not null` : 


```suggestion
    op.add_column("boost_cinema_details", sa.Column("username", sa.TEXT(), autoincrement=False, nullable=True))
    op.add_column("boost_cinema_details", sa.Column("password", sa.TEXT(), autoincrement=False, nullable=True))
```